frontend/src/app/features/details/summary/SummaryInfo.tsx

```diff
@@ -33,14 +50,57 @@ const SummaryInfo: React.FC<ISummaryInfo> = ({
       console.log('Approve Handler Not Provided');
     });
 
+  const CENTER_OF_BC: LatLngTuple = [53.7267, -127.6476];
+
+  const { data } = useMapSearch_FindSiteBySiteIdQuery({
+    variables: { siteId: siteData.id },
+  });
```

Comment:
Doing another request seems a bit wasteful here, we already have access to `siteData` prop, which I assume has the lat/deg data you need (and if it doesn't, you should add those fields to the query)